### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by adanlbenito**

### CODE REVIEW

File: dev/src/example_except.json [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+			"section": "03-Analog",
+			"name": "*"
+		},
+		{
+			"section": "05-Communication",
+			"name": "MIDI"
+		},
+		{
+			"section": "02-Digital",
+			"name": "bicolor-LEDs"
```
[the remark this answers is not in the document]
Comment (adanlbenito) [reply dev/src/example_except.json]: uses analogRead to change the brightness

[the remark this answers is not in the document]
Comment (giuliomoro) [reply dev/src/example_except.json]: Ok, I will fix that (defaulting to max brightness if no analog in). Remove it from this list.

File: IDE/public/example_except.json [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+			"section": "10-Instruments",
+			"name": "d-box"
+		},
+		{
+			"section": "11-Extras",
+			"name": "gpioAnalogLoopbackTest"
+		},
+		{
+			"section": "11-Extras",
+			"name": "salt-factory-test"
```
Comment (giuliomoro) [inline IDE/public/example_except.json]: taste ?

Comment (adanlbenito) [reply IDE/public/example_except.json]: this is what happens when you put funny names to examples... I'll fix it now

File: dev/src/Views/SettingsView.js [hunk 3]
```diff
@@ -325,30 +325,39 @@ class SettingsView extends View {
...
+				sections: ['capelet-settings'],
+				subsections: ['mute-speaker']
+			},
+			Ctag: {
+				sections: ['capelet-settings'],
+				subsections: ['disable-led', 'mute-speaker', 'hp-level', 'pga-left', 'pga-right']
+			},
+			CtagOnly: {
+				sections: ['capelet-settings'],
+				subsections: ['analog-channels', 'analog-samplerate', 'use-analog', 'dac-level']
```
Comment (giuliomoro) [inline dev/src/Views/SettingsView.js]: the DAC level is on the CTAG, the ADC is not.

File: dev/src/Views/TabView.js [hunk 4]
```diff
@@ -4,16 +4,16 @@ var View = require('./View');
 var _tabsOpen = false;
 
 class TabView extends View {
-
```
Comment (giuliomoro) [inline dev/src/Views/TabView.js]: @adanlbenito again these whitespaces? Is that your editor?

**Pull request merged**